README.md

```diff
@@ -80,7 +80,10 @@ Lnd uses the `P-521` curve for its certificates but NodeJS gRPC module is only c
 You need to generate your own lnd certificates using the following commands (thanks to Alex Akselrod for helping me on this):
 
 ```
+# Enter the Lnd home directory, located at ~/.lnd on Linux or 
+# /Users/[username]/Library/Application Support/Lnd/ on Mac OSX
 cd ~/.lnd
+/Users/[username]/Library/Application Support/Lnd/
```

Comment:
this line looks strange to, may be me should add a `cd` in front of it or comment it ? what do you think ?